main.go

```diff
@@ -199,6 +199,16 @@ func handleCourseRequests(router *mux.Router) {
 }
 
 func handleScheduleRequests(router *mux.Router) {
+
+	// Past Schedules CRUD Operations
```

Comment:
Might need to go through, fix any conflicts and look at the code in dev. This PR seems to have some overlap with a PR that I put up yesterday and was just merged.

Reply:
@AnthonyRodway done. I fixed the conflicts